kayak_core/src/context.rs

```diff
@@ -536,6 +585,13 @@ impl KayakContext {
         }
     }
 
+    /// Get the last calculated mouse position.
+    ///
+    /// Calling this from a widget will return the last mouse position at the time the widget was rendered.
+    pub fn last_mouse_position(&self) -> (f32, f32) {
+        self.last_mouse_position
+    }
```

Comment:
Should this be exposed? I added it so that the tooltips could access the cursor position when they re-render.

I would put this data on hover, but it doesn't work well with the current event system sine `EventType::Hover {pos: (123.0, 123.0)}` is different from `EventType::Hover {pos: (456.0, 456.0)}` And we need those to be hashable so we can look them up relatively easily.

Reply:
I think doing it this way is fine for now. 👍